bin/ci-release.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+echo "ADDING NPM TOKEN"
+echo "//registry.npmjs.org/:_authToken=\${NPM_AUTH_TOKEN}" > .npmrc
+echo ""
+echo "CHECK AUTHENTICATED USER"
+npm whoami --registry https://registry.npmjs.org
+
+
+if [ ! $? -eq 0 ]; then
+    echo
+    echo "USER AUTHENTICATION FAILED"
+    echo
+    exit 1
+fi
+
+if [ "$TRAVIS_BRANCH" = "master" ]; then
+  # RELEASE AS @NEXT
+  yarn install && \
+    yarn lerna publish --skip-git --exact --force-publish=* --canary=commit --npm-tag=next --registry https://registry.npmjs.org
```

Comment:
I am not sure about `--exact`. There is nothing mentioned in docs https://lernajs.io/#command-publish

Reply:
Melody uses 2.x version of Lerna still.  https://github.com/lerna/lerna/tree/2.x#--exact